cookbooks/aws-parallelcluster-install/recipes/base.rb

```diff
@@ -67,14 +71,19 @@
 include_recipe "aws-parallelcluster-install::python"
 
 # Install lots of packages
-package node['cluster']['base_packages'] do
-  retries 10
-  retry_delay 5
-end
-
-# In the case of AL2, there are more packages to install via extras
-node['cluster']['alinux_extras']&.each do |topic|
-  alinux_extras_topic topic
+if platform_family?('rhel', 'fedora', 'amazon')
+  package node['cluster']['base_packages'] do
+    retries 10
+    retry_delay 5
+    flush_cache({ before: true })
```

Comment:
Is it possible to have just one package resource with conditional flush_cache? Something like
```
package node['cluster']['base_packages'] do
    retries 10
    retry_delay 5
    flush_cache({ before: true }) if platform_family?('rhel', 'amazon')
end
```

Reply:
Thanks it worked